CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+3.1.0
+-----
+
+- Introduce `batchSize` config key (default: 500) - PR [#208](https://github.com/algolia/search-bundle/pull/203)
+    
+    This config allow you craete smaller or later batch call Algolia. The same config is used for the doctrine in the ImportCommand
```

Comment:
later -> bigger
craete -> create
the doctrine -> doctrine

Reply:
Thanks! Updated all the typos.